sdk/guides/convo-persistence.mdx

```diff
@@ -180,6 +180,22 @@ The conversation state includes comprehensive information that allows seamless r
 
 For the complete implementation details, see the [ConversationState class](https://github.com/OpenHands/software-agent-sdk/blob/main/openhands-sdk/openhands/sdk/conversation/state.py) in the source code.
 
+### Persistence Directory Structure
+
+When you set a `persistence_dir`, your conversation will be persisted to a directory containing JSON files that mirror the `trajectory.json` file structure from OpenHands V0. The structure follows the pattern `.conversations/<conversation-id>/`, where each conversation has its own subdirectory.
+
+**Directory structure:**
+```
+.conversations/
+├── <conversation-id-1>/
+│   └── state.json
+├── <conversation-id-2>/
+│   └── state.json
+└── ...
+```
+
+Each `state.json` file contains the serialized conversation state with all events, messages, and metadata - similar to how `trajectory.json` worked in V0. This makes it easy to inspect, debug, or migrate conversation data between systems.
```

Comment:
@Openhands this is wrong. Please review https://github.com/OpenHands/software-agent-sdk/blob/main/openhands-sdk/openhands/sdk/conversation/state.py for more accurate folder structure